Week3/More JavaScript.js

```diff
@@ -58,15 +75,20 @@ console.log(3 === 3 ? "yes" : "no");
     //so that vehicle("blue", 1, 5) prints 'a blue used car'
 
 function vehicle(color, category, age) {
+  if (category > 3) {
+    console.log('Please enter valid number for category of vehicle');
+  } else {
     if (age >= 1) {
-        age = "used";
+      age = 'used';
     } else {
-        age = "new";
+      age = 'new';
     }
-    console.log("a " + color + " " + age + " " + vehicleCategory[category]);
-} 
-    
-vehicle("blue", 1, 5);
+    console.log('a ' + color + ' ' + age + ' ' + vehicleCategory[category]);
+  }
+}
+
+vehicle('blue', 1, 5);
```

Comment:
Will it work if you do:
`vehicle('blue', 3, 5);`
?

Reply:
it will be undefined result because there are just 2 category defined. 
So I changed the "category > 3" to "category > 2".

Reply:
👍